website/docs/api/cli.mdx

```diff
@@ -1699,6 +1702,44 @@ $ python -m spacy project dvc [project_dir] [workflow] [--force] [--verbose] [--
 | `--help`, `-h`    | Show help message and available arguments. ~~bool (flag)~~                                                    |
 | **CREATES**       | A `dvc.yaml` file in the project directory, based on the steps defined in the given workflow.                 |
 
+## distill {id="distill", tag="experimental", version="4"}
+
+Distill a _student_ pipeline from a _teacher_ pipeline. Distillation trains the
+models in the student pipeline on the activations of the teacher's models. A
+typical use case for distillation is to extract a smaller, more performant model
+from large high-accuracy model. Since distillation uses the activations of the
+teacher, distillation can be performed on a corpus without (gold standard)
+annotations.
+
+`distill` will save out the best model from all epochs, as well as the final
+pipeline. The `--code` argument can be used to provide a Python file that's
+imported before the training process starts. This lets you register
+[custom functions](/usage/training#custom-functions) and architectures and refer
+to them in your config, all while still using spaCy's built-in `train` workflow.
+If you need to manage complex multi-step training workflows, check out the new
+[spaCy projects](/usage/projects).
+
+> #### Example
+>
+> ```bash
+> $ python -m spacy distill teacher-pipeline student.cfg --output ./output --paths.distill ./distill --paths.dev ./dev
+> ```
+
+```bash
+$ python -m spacy distill [teacher_model] [student_config_path] [--output] [--code] [--verbose] [--gpu-id] [overrides]
+```
+
+| Name                  | Description                                                                                                                                                                                            |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `teacher_model`       | The teacher pipeline to distill the student from. ~~Path (positional)~~                                                                                                                                |
+| `student_config_path` | The configuration of the student pipeline. ~~Path (positional)~~                                                                                                                                       |
+| `--output`, `-o`      | Directory to store the distilled pipeline in. Will be created if it doesn't exist. ~~Optional[Path] \(option)~~                                                                                        |
```

Comment:
And what happens when it's `None`?

Reply:
Added a remark that no pipeline is saved when this option is absent.